src/avram/where.cr

```diff
@@ -38,7 +38,7 @@ module Avram::Where
   end
 
   abstract class ValueHoldingSqlClause < SqlClause
-    getter value : String | Array(String) | Array(Int32) | Nil
```

Comment:
All because of this... I'm still not sure why this messes with Lucky Params... 😝